test/e2e/framework/framework.go

```diff
@@ -161,28 +191,51 @@ func (f *Framework) GetS3CredentialsFile() []byte {
 	return TestContext.S3CredentialsFile
 }
 
-func (f *Framework) defaultCluster() *Cluster {
+func (f *Framework) GetDefaultCluster() *Cluster {
```

Comment:
why make this public? Iirc the idea was that the default cluster was only there for backwards compatibility and the users of the framework were supposed to interact with clusters explicitly (or through framework as is)

Reply:
if you want to collect in the middle of a run you need to find the appropriate artifacts dir

Reply:
see
https://github.com/scylladb/scylla-operator/pull/2091/files#diff-edab1e2b6ade28c3d773100f49b24e2bba8385435ec181b0b7857f0a5d790dacR260

Reply:
Ok, but you can already do this with `f.Cluster(0)`, The concept of a 'default cluster' was something we wanted to get away from, not further solidify it.

Reply:
This issue is that one of those clusters is still bound to the ginkgo which is the reason it still has a special meaning. I am not a fan of using a magic index when you don't choose from an array.

Reply:
Also the object were created with the "default" client - `f.ScyllaAdminClient()` delegates through `GetDefaultCluster` and user didn't choose the index on that call, there should be a corresponding way to get the artifacts dir for it, also without knowing the magic index.

Reply:
My point is mainly to discourage the use of a "default cluster" in the actual test specs. Maybe calling it a `GinkgoCluster` would make sense then? Or anything that would make it clear it's related to test infra, and you're already using a `ginkgoNamespace` across this PR.

Reply:
it has crossed my mind as well but didn't pass a test of uniformity on the naming of how the objects get created. I've wired a special method instead, similar to the namespace.

Reply:
I think that's better, thanks